microbial-cell.csl

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="expanded" default-locale="en-US">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Microbial Cell</title>
+    <title-short>MIC</title-short>
+    <id>http://www.zotero.org/styles/microbial-cell</id>
+    <link rel="self" href="http://www.zotero.org/styles/microbial-cell"/>
+    <author>
+      <name>Didac Carmona</name>
+    </author>
+    <author>
+      <name>Stefan Benke</name>
+    </author>
+    <author>
+      <name>Frank Madeo</name>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="science"/>
+    <updated>2014-02-24T18:25:46+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+    <link/>
+    <link/>
```

Comment:
Please make sure your style follows our [style requirements](https://github.com/citation-style-language/styles/wiki/Style-Requirements). In particular, the style needs to validate, and we need a "documentation" and a "template" link.